webviz_subsurface/plugins/_well_cross_section.py

```diff
@@ -19,6 +20,10 @@
 from .._datainput.surface import load_surface
 
 
+@deprecated_plugin(  # type: ignore
+    "This plugin will soon be deprecated. "
+    "Relevant functionality is implemented in the StructuralUncertainty plugin."
+)
```

Comment:
I think `@deprecated_plugin` takes in two arguments (one short string + one longer explanation). Here it is one string only? Maybe that is also why mypy complains?

https://github.com/equinor/webviz-config/blob/master/CONTRIBUTING.md#deprecate-plugins-or-arguments